scripts/convert_gen2_to_gen3_hsc.sh

```diff
@@ -8,7 +8,30 @@
 
 # Run the gen2 to gen3 conversion code
 # This conversion is done in-place so the same path is used for the gen2 and gen3 repos.
-butler convert hsc -i lsst.obs.subaru.HyperSuprimeCam --gen2root ./hsc
+
+# Clean out old gen3 repo if it is there
+if [ -f "hsc/butler.yaml" ]; then
+    echo "Clearing out old butler.yaml"
+    rm hsc/butler.yaml
+fi
+
+if [ -f "hsc/gen3.sqlite3" ]; then
+    echo "Clearing out old gen3.sqlite3"
+    rm hsc/gen3.sqlite3
+fi
+
+if [ -d "hsc/HSC/calib" ]; then
+    echo "Clearing out old curated calibrations"
+    rm -r hsc/HSC/calib
+fi
+
+# Do the basic conversion
+butler convert hsc --gen2root ./hsc -C scripts/config/convertRepoHsc.py
+# Bring in the curated calibrations
+butler write-curated-calibrations hsc -i HSC
```

Comment:
I don't understand how this works. Have you disabled writing curated calibrations in butler convert? By default butler convert will do the write curated calibrations. I'm confused because I expect this command to fail telling you that the curated calibrations are already present.

Reply:
As far as I know, `butler convert` does _not_ do the write curated calibrations.  I think we had this discussion before? https://lsstc.slack.com/archives/C3UCAEW3D/p1594336746279600

Reply:
It does though. Except now that I think about it, it only does that if there are calibrations in the input gen2 repo. That is a hilarious oversight of course. That has to be a bug. I don't understand how that makes any sense in retrospect.

```
$ butler  convert yyy --gen2root ../obs_lsst/data/input/ts8 --calibs=CALIB
$ $ ls yyy/LSST-TS8/calib/
bias		camera		qe_curve
```